cmd/addexport.go

```diff
@@ -133,6 +143,40 @@ func (p *AddExportParams) PreInteractive(ctx ActionCtx) error {
 		return err
 	}
 
+	if p.export.IsService() {
+		ok, err := cli.PromptBoolean("track service latency", false)
+		if err != nil {
+			return err
+		}
+		if ok {
+			_, err = cli.Prompt("sampling frequency percentage [1-100]", "", false, func(s string) error {
+				v, err := strconv.Atoi(s)
+				if err != nil {
+					return err
+				}
+				if v < 1 || v > 100 {
```

Comment:
If you wanted to here you could accept 0 to turn off. Not critical but you and I discussed.

Reply:
ok - let's do that.